saleor/graphql/order/types.py

```diff
@@ -212,7 +212,9 @@ def _resolve_user(event_user):
     @staticmethod
     def resolve_app(root: models.OrderEvent, info):
         requestor = get_user_or_app_from_context(info.context)
-        if requestor_has_access(requestor, root.user, AppPermission.MANAGE_APPS):
+        if requestor_has_access(
+            requestor, root.user, AppPermission.MANAGE_APPS
+        ) | requestor_has_access(requestor, root.user, OrderPermissions.MANAGE_ORDERS):
```

Comment:
This will give access to the entire App object to a user with `MANAGE_ORDERS`. Instead, we should allow the "order manager" to query apps from order events, but limit the fields they can ask for by adding permission checks inside the `App` type. I think order manager (or even all staff users) should have access to fields like `name`, `appUrl`, `homepageUrl`, `supportUrl`, `aboutApp`. Fields like `permissions`, `tokens`, `webhooks` etc should require `MANAGE_APPS` permission.

Reply:
@patrys Do you agree? We should do the same for `Order.user` field.